sample-windows.ini

```diff
@@ -0,0 +1,45 @@
+[General]
+
+hostname = 0.0.0.0
+
+portNumber = 8000
+
+; If any symbols of interest aren't available locally (e.g. Windows DLLs), uncomment line below
+
+remoteSymbolServer = http://symbolapi.mozilla.org:80/
+
+
+
+; These options are unlikely to need editing by users
+
+maxCacheEntries = 2000000
+
+mruSymbolStateFile = /tmp/snappy-mru-symbols.json
+maxMRUSymbolsPersist = 10
+
+[Log]
+
+maxFiles = 10
+
+maxFileSize = 1048576
+
+;logPath = /tmp/log/snappy
```

Comment:
put it somewhere visible.. maybe %HOME% if python supports the escapes, otherwise hard-code windows temp directory